betty/project/extension/demo/__init__.py

```diff
@@ -79,9 +81,12 @@ def comes_after(cls) -> set[PluginIdentifier[Extension]]:
         return cls.depends_on()
 
     @override
-    def register_event_handlers(self, registry: EventHandlerRegistry) -> None:
-        registry.add_handler(
-            LoadAncestryEvent, lambda event: load_ancestry(event.project)
+    async def load(self, job: Job[ProjectContext]) -> None:
+        await job.scheduler.add(
+            load_ancestry,
+            DemoJobNames.LOAD,
+            dependencies={LoadJobNames.PRE_LOAD},
```

Comment:
Can we make this pattern easier and more straightforward, where a job comes after the pre job and before the post job?